corehq/apps/locations/forms.py

```diff
@@ -243,28 +242,35 @@ def clean_site_code(self):
 
         return site_code
 
+    def _get_allowed_types(self, parent):
```

Comment:
nit picky but might be easier if this took a `domain` and a `parent` and then got pulled out of the class or made static